src/design-tokens/tier-1-definitions/typography-presets.css

```diff
@@ -120,10 +119,9 @@
 @define-mixin eds-typography-preset-9 $important {
   font-family: var(--eds-font-family-tertiary) $important;
   font-weight: var(--eds-font-weight-regular) $important;
-  font-size: var(--eds-font-size-md-2) $important;
-  line-height: var(--eds-line-height-sm) $important;
-  letter-spacing: 1.5px $important;
-  text-transform: uppercase $important;
+  font-size: var(--eds-font-size-sm) $important;
```

Comment:
kinda weird that `preset-9` is larger than `preset-8` ... I think we want them to be in descending size

Reply:
Oops I didn't realize that, thanks!